Networking/WebSockets/BuiltInWebSocket.cc

```diff
@@ -103,9 +103,8 @@ namespace litecore { namespace websocket {
 
     void BuiltInWebSocket::closeSocket() {
         logVerbose("closeSocket");
-        if (_socket &&_socket->connected()) {
-            _socket->close();   // This will often be detected in readFromSocket(), which will close it
-            _socket->interrupt(); // But on Windows it usually is not, so interrupt it to poke it into doing the right thing
+        if (_socket) {
```

Comment:
Note: The `connected()` test and `interrupt()` were moved into `TCPSocket::close`, so the behavior remains the same.